tailwind.config.js

```diff
@@ -58,6 +58,9 @@ module.exports = {
         8: '1.938rem',
         11: '3.188rem',
       },
+      fontSize: {
+        '4.5xl': '2rem',
```

Comment:
That's probably `3.5xl`
![image](https://user-images.githubusercontent.com/49658988/189194259-a0b5429f-44fb-48f4-80ba-e24a9351a3b6.png)

Reply:
Btw, you don't use it